azure-devops/ecommerce-projects/03_service_connections_aks.tf

```diff
@@ -0,0 +1,38 @@
+resource "azuredevops_serviceendpoint_kubernetes" "aks_dev" {
+  depends_on            = [data.azuredevops_project.project]
+  project_id            = data.azuredevops_project.project.id
+  service_endpoint_name = local.srv_endpoint_name_aks_dev
+  apiserver_url         = module.ecommerce_dev_secrets.values["pagopa-d-weu-dev-aks-apiserver-url"].value
```

Comment:
not a problem but I noticed that we have inserted a duplication of information in the names `d` and` dev` too, which makes the naming convention a bit verbose